Sources/interoperability-tests/InteroperabilityTestsExecutable.swift

```diff
@@ -60,4 +60,95 @@ struct InteroperabilityTestsExecutable: AsyncParsableCommand {
       }
     }
   }
+
+  struct RunTest: AsyncParsableCommand {
+    static var configuration = CommandConfiguration(
+      abstract: "Runs a gRPC interoperability test using a gRPC Swift client."
+    )
+
+    @Option(help: "The host the server is running on")
+    var host: String
+
+    @Option(help: "The port to connect to")
+    var port: Int
+
+    @Argument(help: "The name of the test to run")
+    var testName: String
+
+    func run() async throws {
+      guard let testCase = InteroperabilityTestCase(rawValue: self.testName) else {
+        throw InteroperabilityTestError.testNotFound(name: self.testName)
+      }
+
+      try await Self.runTest(
+        testCase: testCase,
+        host: self.host,
+        port: self.port
+      )
+    }
+
+    internal static func runTest(
+      testCase: InteroperabilityTestCase,
+      host: String,
+      port: Int
+    ) async throws {
+      let group = MultiThreadedEventLoopGroup(numberOfThreads: 1)
+      defer {
+        try! group.syncShutdownGracefully()
+      }
```

Comment:
No need to do this, just use the default param in the `init` which is the singleton MTELG